src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -14,37 +14,55 @@ redirects:
 
-Our usage-based pricing model is simple, which helps you more easily predict your billing. We bill on these two primary factors:
+## New Relic billing factors [#how-pricing-works]
 
-* Your [ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing). We bill on data ingested, no matter the type, making your billing easy to understand. You'll receive 100 GBs per month for free. The price per GB above the free level varies depending on which [data option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you have: Data Plus or original.
-* Your [billable user count](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing). Core users and full platform users are billable. Basic users are free. Your per-user cost depends on your pricing edition. Free tier and Standard edition get one free full platform user.
+Our usage-based pricing model is simple, which helps you easily predict your billing. We bill on these two primary factors:
+
+* **How much data you ingest**. We bill on data ingested, no matter the type, making your billing easy to understand. You can ingest 100 GBs of data per month for free. The price per GB above the free level varies depending on which data option you have: [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-prices) or original. To learn more about how we bill for ingested data, see [Data ingest billing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing).
+* **How many billable users you have**. There are three types of users. Core users and full platform users are billable, and basic users are free. The cost per full platform user depends on your pricing edition. For free tier and Standard edition, you get one free full platform user. To learn more about how we bill for users, see [User billing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing). 
 
 ### Pricing editions [#editions]
 
-Besides determining the cost of your billable users, the editions (Standard, Pro, Enterprise) gate access to different features and limits (for example, different limits for synthetic monitor checks, and access to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which gives you HIPAA- and FedRAMP-compliant data). For more on editions, see the [pricing page](https://newrelic.com/pricing). 
+Besides determining how much your full platform users cost, your edition (Standard, Pro, or Enterprise) gates access to assorted features and limits (for example, different limits for synthetic monitor checks, and access to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which gives you HIPAA- and FedRAMP-compliant data). For more about edition differences, see our [pricing page](https://newrelic.com/pricing). 
```

Comment:
"gates" seems a bit awkward to read to me. Maybe just "provides" or something like that?